docs/admin/node.md

```diff
@@ -247,6 +248,12 @@ Set the `cpu` and `memory` values to the amount of resources you want to reserve
 Place the file in the manifest directory (`--config=DIR` flag of kubelet).  Do this
 on each kubelet where you want to reserve resources.
 
+## API Object
+
+Node is a top-level resource in the kubernetes REST API. More details about the
+API object can be found at: [Node API
+object](https://htmlpreview.github.io/?https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-reference/definitions.html#_v1_node).
```

Comment:
I'm not sure what the best way is to link to these docs.

We're working to get them on kubernetes.io. However, the standard approach of using a relative link won't work since github won't render html. Similarly, the releases.k8s.io/HEAD redirector won't work, either.

I think we're going to need to link to the versioned docs in kubernetes.io and rewrite the links similar to how we rewrite the redirector links.

@caesarxuchao @thockin